src/components/stimuli/Stimuli.tsx

```diff
@@ -1,6 +1,11 @@
 import React, { useEffect, useState } from 'react';
+import Spinner from 'react-bootstrap/Spinner';
+import Toast from 'react-bootstrap/Toast';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
 import StimuliCard from './stimuliCard';
 
 const generateCards = (stimuliUrls: Array<string>): Array<JSX.Element> => {
```

Comment:
You should probably put changes to Stimuli in your open PR that's related to Stimuli